app/models/concerns/pick_up_order.rb

```diff
@@ -0,0 +1,89 @@
+class PickUpOrder < ActiveRecord::Base
+  # rubocop:disable Metrics/BlockLength
```

Comment:
I don't think we specifically added it but it's a default. I propose we make a westernmilling/linter-config repo to be the agreed upon Rubocop and other litner settings and then we can just copy and paste those to each project (or maybe use git submodules? Those may be more pain than help though)